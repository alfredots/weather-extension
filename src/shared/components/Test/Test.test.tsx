import { describe, it, expect } from 'vitest';
import { render, screen } from '@testing-library/react';
import { Test } from './Test';

describe('Test', () => {
  it('renders the App component', () => {
    const { container } = render(<Test />);

    expect(container.firstChild).toMatchSnapshot();
    screen.debug(); // prints out the jsx in the App component unto the command line
  });
});
