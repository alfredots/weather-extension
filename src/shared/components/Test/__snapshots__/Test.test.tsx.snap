// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test > renders the App component 1`] = `
.c0 {
  color: red;
  background-color: aliceblue;
  font-size: 12px;
}

<div
  class="c0"
>
  Teste
</div>
`;
